microsoft-edge/devtools-guide-chromium/console/console-dom-interaction.md

```diff
@@ -5,290 +5,345 @@ author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.prod: microsoft-edge
-ms.date: 04/13/2021
+ms.date: 10/18/2023
 ---
 # Interact with the DOM using the Console
 
-The **Console** tool is a great way to interact with the webpage in the browser.<!-- todo: add intro explanation -->  The **Console** is like a script-environment version of the [Inspect tool](../css/inspect.md).<!-- todo: add intro explanation -->
+The **Console** tool is a great way to interact with the webpage in the browser. In DevTools, you usually interact with the DOM by using the **Inspect** tool to select elements, and by using the **Elements** tool to modify the DOM, for example to add or change element attributes or styles.
+
+The **Console** tool can also be used to interact with the DOM by using JavaScript code. For example, you can find elements in the DOM tree and manipulate them by using DOM APIs.
+
+To learn more about the **Inspect** tool and the **Elements** tool, see [Analyze pages using the Inspect tool](../css/inspect.md) and [Inspect, edit, and debug HTML and CSS with the Elements tool](../elements-tool/elements-tool.md).
```

Comment:
```suggestion
To learn more about the **Inspect** tool, see [Analyze pages using the Inspect tool](../css/inspect.md). To learn more about the **Elements** tool, see [Inspect, edit, and debug HTML and CSS with the Elements tool](../elements-tool/elements-tool.md).
```
1 idea at a time instead of ideas A B A B intertwined.  Acrolinx might suggest breaking up sentence.